scripts/synthetix/facilitate_trade.js

```diff
@@ -73,44 +88,89 @@ module.exports = async (callback) => {
     // Note that sUSD always has a price of 1 within synthetix protocol.
     const exchangeRate = await snxjs.ExchangeRates.rateForCurrency(sETHKey)
     const formatedRate = snxjs.utils.formatEther(exchangeRate)
-    console.log("sETH Price", snxjs.utils.formatEther(exchangeRate))
+    console.log("Oracle sETH Price (in sUSD)", formatedRate)
+
+    const exchangeBuyETHPrice = await estimatePrice(
```

Comment:
The name of the variable sounds to me as "the price at which the exchange *buys* ETH", but this is more "the price at which the exchange *lets you buy* ETH". What about `exchangePriceWhenBuyingUsdForEth`? Not nice, I admit, but I hope clearer. Same thing for the other exchange price. It should also be clearer that the price is in ETH, so 1/what you are normally used to.